.github/workflows/build-macos.yml

```diff
@@ -39,3 +39,38 @@ jobs:
         with:
           name: node${{ matrix.target-node }}-macos-x64
           path: dist/*
+
+  macos-arm64:
+    runs-on: macos-11.0
+
+    strategy:
+      fail-fast: false
+      matrix:
+        target-node: [14, 16, 18, 19]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: latest
+
+      - name: Use Node.js 14
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - run: yarn install --ignore-engines
+
+      - run: yarn start --node-range node${{ matrix.target-node }} --arch arm64 --output dist
```

Comment:
How does this work? Is it actually compiling the node binary?

Reply:
Yeah that's what the scripts in this package do. As far as I remember.

Reply:
Is that script coming from the https://github.com/actions/setup-node/tree/main/src. I'm not super familiar with GitHub Actions, where can I find the source for what the actual script is?

Reply:
e.g. https://github.com/yao-pkg/pkg-fetch/blob/main/lib/build.ts, with the actual entry point defined in package.json.

This entire repo is essentially a script/package to build and fetch patched node binaries suitable for pkg.